Userland/Libraries/LibGUI/WordWrapMode.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021-2021, Vinicius Sugimoto <vtmsugimoto@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Optional.h>
+#include <AK/StringView.h>
+
+namespace GUI {
+
+#define GUI_ENUMERATE_WORD_WRAP_MODES(M) \
+    M(Word)                              \
+    M(Anywhere)
```

Comment:
I know we do this in a couple of places, but is it *really* necessary for a two-member enum to use convenience enumeration macros? I'd like to hear what the maintainers say about this.